### PR TITLE
Remove @brief in DBA.php

### PR DESCRIPTION
**Pull request by annando**
I think that most occurrences of `@brief` aren't needed. The changes in this file are some kind of "test balloon" to ensure that further work on the more than 100 files who still have `@brief` elements will not be useless.

### CODE REVIEW

File: src/Database/DBA.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public static function getConnection()
 	}
 
 	/**
-	 * @brief Returns the MySQL server version string
```
Comment (MrPetovan) [inline src/Database/DBA.php]: This doc block already had a long description making the `@brief` tag warranted.

Comment (annando) [reply src/Database/DBA.php]: But the text below wasn't the long form of the `@brief` part.

Comment (MrPetovan) [reply src/Database/DBA.php]: Fair enough.

Comment (nupplaphil) [reply src/Database/DBA.php]: Where does `@brief` come from? At least PHPStorm doesn't know what to do with it. I found the "official" way of separating summary from description here: https://docs.phpdoc.org/latest/guides/docblocks.html#summary ... Do we really need this tag at all?

Comment (MrPetovan) [reply src/Database/DBA.php]: @rabuzarus introduced it as a summary.

**Pull request merged**